acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/SigningAcceptanceTestBase.java

```diff
@@ -51,6 +51,15 @@ protected void setupEth2Signer(final Eth2Network eth2Network, final SpecMileston
     startSigner(builder.build());
   }
 
+  protected void setupEth2SignerWithCustomNetworkConfig(final String networkConfigFile) {
+    final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
+    builder.withKeyStoreDirectory(testDirectory).withMode("eth2").withNetwork(networkConfigFile);
```

Comment:
withNetwork looks like it should receive the network name, e.g. "minimal" rather than the config file path...I guess this is having no impact on the test though and so maybe the param isn't needed at all here?

Reply:
The `--network` flag takes both predefined network names as well as config file paths. Since this is our AT DSL, I think it is safe to rename `withNetwork` to `withNetworkNameOrConfigFile`.

Reply:
I created overridden method for withNetwork. One takes `String`, other takes `Path`.

Reply:
got it - looks like that's behaviour inherited from teku?

Reply:
yes, that is correct, we are using same semantics (and internal libraries) as Teku.